pkg/pcap/pcaps.go

```diff
@@ -0,0 +1,95 @@
+package pcap
```

Comment:
nit: since the "main" module struct is here rather than in `pcap.go` maybe the package should be `pcaps` instead.
Then `NewPcaps` can just be `New` as importers will use it as `pcaps.New()`.

Reply:
Yep, I thought about it before, will do.